helm/cluster-openstack/templates/open_stack_cluster_template.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+kind: OpenStackClusterTemplate
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: {{ include "resource.default.name" . }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  template:
+    spec:
+      cloudName: {{ .Values.cloudName }}
+      identityRef:
+        name: {{ .Values.cloudConfig }}
+        kind: Secret
+      managedAPIServerLoadBalancer: true
+      managedSecurityGroups: true
+      {{- if .Values.nodeCIDR }}
+      nodeCidr: {{ .Values.nodeCIDR }}
+      {{- end }}
+      {{- if .Values.externalNetworkID }}
```

Comment:
Is there a viable scenario where this can be empty?
Maybe it would be better to fail here if there isn't?

Reply:
Yes. It's only required for installations with multiple external networks. In simple installations there is only one and it would therefore be optional.